lib/src/components/layout/accordion.dart

```diff
@@ -70,13 +180,13 @@ class _AccordionItemState extends State<AccordionItem>
     _expanded.value = widget.expanded;
     _controller = AnimationController(
       vsync: this,
-      duration: const Duration(milliseconds: 200),
+      duration: widget.theme.duration,
       value: _expanded.value ? 1 : 0,
     );
     _easeInAnimation = CurvedAnimation(
       parent: _controller,
-      curve: Curves.easeIn,
-      reverseCurve: Curves.easeOut,
+      curve: widget.theme.curve,
+      reverseCurve: widget.theme.reverseCurve,
```

Comment:
Do not replace the default value from the theme, as the theme should not hold the default value.
```dart
value: fromTheme ?? defaultValue,
```